pkg/execution/executor/executor.go

```diff
@@ -1717,17 +1720,42 @@ func (e *executor) Resume(ctx context.Context, pause state.Pause, r execution.Re
 						attribute.Bool(consts.OtelSysStepInvokeExpired, r.EventID == nil),
 					),
 				)
+				span.Send()
+				defer span.End()
+				span.SetAttributes(commonAttrs...)
```

Comment:
Could we be _intensely_ unlucky here and export the span before the second call to `SetAttributes()`?

Reply:
addressed in #1360